### ISSUE TITLE
Regression.py has errors

### ISSUE THREAD

**Issue by PushTheGeek** [babarishith/LARA#1]
the "s_d" is still present in the regression.py at line 24.  when i run it, it says "s_d undefined".
I tried to replace it with constant "5", D, and K, and i get some tensorflow error.

**Comment by CaptainSharf** [comment]
Can you give me some time.I will sort it out

**Comment by nekmar** [comment]
Eventually did you figure out what s_d  is?

**Comment by Efthimis13** [comment]
Hello Nekmar,

May I ask you something about the aspect_segmentation code. Did you find all the aspect_keywords.csv as the example has. I found only few of the words. 

Apart from that my output sometimes have values bigger than 5 see an example:
aspect_ratings=[array([ 4., 0., 5., 0., 0., 4., 0.]), array([ 3., 0., 0., 0., 0., 0., 0.]), array([ 6., 0., 5., 0., 0., 0., 0.]),
It seems that my values from the previous stage are not between [-1,1]
```
# Aspect Ratings Per Review
    for j, l in enumerate(r):
        for k in range(7):
            if k in l:
                rating[k] += rs[j]
                print('rating',rating)
```

output: rating= [ 1.4729 0.8822 0.7184 0. 0.8822 0.8822 0. ] (The first value is bigger than 1)

Thank you for your time

p.s.: Is it possible share with me your code or is it possible to send you mine

Efthimis

**Comment by nekmar** [comment]
The problem is that I have an issue with the s_d. Have you figured out what this is.
p.s. if you want you can send me a message or email me.

**Comment by Efthimis13** [comment]
Hello Nekmar,

I haven't figured out yet.
I have to fix first the above problems with the aspect_segmentation code. Does your code(aspect_segmentation)    work OK? Is it possible to send me your e-mail address? 

thank you

Efthimis

**Comment by nekmar** [comment]
Hey, send me at nek.mar93@gmail.com
Thanks

2017-10-26 9:46 GMT+03:00 Efthimis13 <notifications@github.com>:

**Comment by Pragyanand** [comment]
Is the problem with s_d has been fixed yet or any solution, if anyone has found something to make the code work?

**Comment by redris96** [comment]
We initially tried to code the paper as is it and hence tried to use regression. But it did not work properly so we ended using the sentiment analysis of nltk to score individual statements and not the regression. You can use the aspect_spegmentation.py code, it's self contained and does not depend on regression. 
If any of you can make the regression work and contribute to the project it would be great. :)

**Comment by Pragyanand** [comment]
Thank you so much for replying. I really am a beginner and wanted to know if this code works because i tried everything i knew about it.
Now, should i leave the regression part as is and use aspect_segmentation instead and does it work?

**Comment by redris96** [comment]
Yup, just forget about regression.py and use aspect_segmentation.py

**Comment by redris96** [comment]
I have updated the Readme with the same.

**Comment by Pragyanand** [comment]
Thank you for that and i have a query about how the aspect_keywords file was created?
